guides/hardware/opening-up-kuri.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: guide
+title: Opening Up Kuri's Torso
+category: hardware
+tags:
+- disassembly
+---
+
+## Goal
+Remove Kuri's front shell so that you can get at her main board.
+
+## Overview
+To get at the inside of Kuri's torso, we will remove both arm plates, unscrew
+the screws underneath, then pop the front shells off and unhook the cables
+connecting the shells to the rest of Kuri.
+
+## Removing Kuri's arm plates
```

Comment:
Any chance we could add some photos or drawings to these steps? Maybe it'd be worthwhile to PNG-ize and embed one of the subassembly drawings here

Reply:
I can add photos later this week or next week, but as people are trying to onboard their Kuris as we speak and many of them are failing at it for the exact same reason, I would like to get these two PRs merged ASAP so I can email them back with instructions that are actually in the documentation.  Or I can take out the reference to this one, and just merge the other.